storage/replica.go

```diff
@@ -1433,12 +1436,20 @@ func (r *Replica) handleRaftReady() error {
 func (r *Replica) tick() error {
 	r.mu.Lock()
 	defer r.mu.Unlock()
+	r.mu.ticks++
 	r.mu.raftGroup.Tick()
-	// TODO(tamird/bdarnell): Reproposals should occur less frequently than
-	// ticks, but this is acceptable for now.
-	// TODO(tamird/bdarnell): Add unit tests.
-	err := r.reproposePendingCmdsLocked()
-	return err
```

Comment:
LGTM

On Mon, May 2, 2016 at 11:32 AM marc <notifications@github.com> wrote:

> In storage/replica.go
> <https://github.com/cockroachdb/cockroach/pull/6448#discussion_r61754922>:
>
> >  	r.mu.raftGroup.Tick()
> > -	// TODO(tamird/bdarnell): Reproposals should occur less frequently than
> > -	// ticks, but this is acceptable for now.
> > -	// TODO(tamird/bdarnell): Add unit tests.
> > -	err := r.reproposePendingCmdsLocked()
> > -	return err
>
> any reason to keep incrementing vs resetting when larger than
> RaftElectionTimeoutTicks?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/6448/files/bd314346b9950044b5e80c84cbf518646d7a19c4#r61754922>
>
-- 

-- Tobias